content/en/tutorials/lotus/build-with-lotus-api.md

```diff
@@ -25,6 +25,7 @@ In this tutorial we'll set up a Lotus node locally and use the filecoin.js libra
 Before getting started, you should be familiar with:
 
 - [How Filecoin works](https://docs.filecoin.io/basics/what-is-filecoin/overview/)
+- [What is Lotus](https://lotus.filecoin.io/lotus/get-started/what-is-lotus/)
```

Comment:
```suggestion
- [What is Lotus]({{< relref "../../lotus/get-started/what-is-lotus/" >}})
```

Changing the to a relref-link to ensure that the link will be properly resolved within the Hugo site structure